README.md

```diff
@@ -244,11 +249,11 @@ await index.updateAttributesForFaceting([
   ])
 ```
 
-You only need to perform this operation once.
+You only need to perform this operation once per index.
 
-Note that Meilisearch will rebuild your index whenever you update `filterableAttributes`. Depending on the size of your dataset, this might take time. You can track the process using the [tasks](https://docs.meilisearch.com/reference/api/tasks.html#get-task)).
+Note that Meilisearch will rebuild your index whenever you update `filterableAttributes`. Depending on the size of your dataset, this might take considerable time. You can track the process using the [tasks API](https://docs.meilisearch.com/reference/api/tasks.html#get-task)).
```

Comment:
```suggestion
Note that Meilisearch rebuilds your index whenever you update `filterableAttributes`. Depending on the size of your dataset, this might take considerable time. You can track the process using the [tasks API](https://docs.meilisearch.com/reference/api/tasks.html#get-task)).
```

Not sure about that

Reply:
Not sure about your edit or about the info?

IMHO, your edit is pretty good and I want to commit it.

Reply:
I wasn't sure if it was correct english